### PR TITLE
Tara UI

### PR DESCRIPTION
**Pull request by tparanjpe**

### CODE REVIEW

**Review by sanghoonsim — APPROVED** [review]

File: application/app/index.js [hunk 1]
```diff
@@ -208,7 +248,7 @@ function transferFile(){
           } 
       }    
   } catch (err) {
-    console.error(err);
+    console.log(err)
```
Comment (aderbique) [inline application/app/index.js:251]: You are missing semi colons on your log statements

File: application/app/index.js [hunk 2]
```diff
@@ -186,9 +226,9 @@ function searchFiles(){
     while ((dirIter = listDir.next()) && !dirIter.done) {
       dirArr.push(dirIter.value);
     }
-    return dirArr;
+    return dirArr    
   } catch (err) {
-    console.log('No files to be uploaded');
+    console.log('No files')
```
Comment (aderbique) [inline application/app/index.js:231]: You are missing semi colons on your log statements

File: application/app/index.js [hunk 3]
```diff
@@ -186,9 +226,9 @@ function searchFiles(){
     while ((dirIter = listDir.next()) && !dirIter.done) {
       dirArr.push(dirIter.value);
     }
-    return dirArr;
+    return dirArr
```
Comment (aderbique) [inline application/app/index.js:229]: Missing semi colon

File: application/app/index.js [hunk 4]
```diff
@@ -252,7 +292,7 @@ it deletes the file in accordance with the flags sent from companion
 */
 messaging.peerSocket.addEventListener("message", (evt) => {
   var files = searchFiles();
-  var filename = files[0];
+  var filename = files[0]
```
Comment (aderbique) [inline application/app/index.js:295]: Missing semi colon

**Review by aderbique — CHANGES_REQUESTED** [review]

**Review by aderbique — APPROVED** [review]

**Comment by aderbique** [comment]
lgtm

**Pull request merged**